src/pages/upload-timetable-page/UploadTimetablePage.tsx

```diff
@@ -53,16 +52,7 @@ export const UploadTimetablePage = ({
           const loadedEvents =
             csvToObjectArray<DevelopmentPlanTimetable>(csvString);
           setLoadedTimetableEvents(loadedEvents);
-          setTimetableEvents(
-            loadedEvents
-              // This assumes any row with an end date is invalid
-              .filter(
-                (event) =>
-                  !event.endDate &&
-                  event.developmentPlanEvent !==
-                    TimetableEventKey.TimetableUpdated
-              )
-          );
+          setTimetableEvents(loadedEvents.filter(isValidEvent));
```

Comment:
probably should move the check of timetable-updated event here: 
`
setTimetableEvents(loadedEvents.filter((event)=> 
isValidEvent(event) && 
event.developmentPlanEvent !==
TimetableEventKey.TimetableUpdated ));
`

Reply:
change applied